Directory.Build.props

```diff
@@ -2,6 +2,7 @@
 <Project>
   <PropertyGroup>
     <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
```

Comment:
Nit:
```suggestion
    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
```

Reply:
We _can_ do that, but coming from the MSBuild team myself (several years ago) we learned it was easier to read the condition expression when the double and single quotes were separated by a space. So I'd _rather_ not make this change.